pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -1050,6 +1053,15 @@ zoneDisallowedNotify OBJECT-TYPE
     DESCRIPTION
         "Number of NOTIFY operations not allowed by allow-notify-for"
     ::= { stats 125 }
+
+nonResolvingNameserverEntries OBJECT-TYPE
+    SYNTAX CounterBasedGauge64
+    MAX-ACCESS read-only
+    STATUS current
+    DESCRIPTION
+        "Number of entries in the non-resolving NS name cache"
+    ::= { stats 126 }
```

Comment:
The new entry (and the two previous ones, by the look of it) should be added to the "Objects conformance group" below.